### ISSUE TITLE
Error Building With Docker

### ISSUE THREAD

**Issue by th5** [carnival-data/carnival-demo-biomedical#9]
I'm getting an error when building under Docker, 
```
A problem occurred configuring root project 'carnival-micronaut'.
The value for task ':test' property 'testFrameworkProperty' is final and cannot be changed any further.
```
I was worried that I accidentally introduced a regression recently, so I tried rolling back to versions right before and after we changed from pmbb-ibi to carnival-data. The version right after gets same error. The version before gets an error about not being able to find carnival at the old URL.

Another idea is that uploads to the Maven servers are goofing things up. However we don't show up at all on search.maven.org (official Maven Central search).

The last idea I had was that Docker images changed under our feet. I'll check out the gradle image we're pulling in.

---

docker-compose build 
or
docker-compose -f docker-compose-test.yml build
```
[CarnivalApplication] Carnival version: 2.1.0

FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'carnival-micronaut'.
> The value for task ':test' property 'testFrameworkProperty' is final and cannot be changed any further.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

See https://docs.gradle.org/7.3/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 23s
ERROR: Service 'web' failed to build: The command '/bin/sh -c gradle shadowJar --no-daemon' returned a non-zero code: 1
```

**Comment by hjwilli** [comment]
Looks like the version of gradle changed under us. Gradle 7.3.0 was released recently, and it seems like an error was introduced between that version and micronaut (see https://github.com/micronaut-projects/micronaut-gradle-plugin/issues/296)

The dockerfile was using gradle image tagged `jdk11` so it was using latest gradle version on JDK 11. I changed it to `7.2.0-jdk11` and that seemed to resolve the issue.

https://hub.docker.com/_/gradle
